document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/ProsecutionType.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.document.generator.prosecution;
+
+public enum ProsecutionType {
+
+    ULTIMATUM("prosecution", "ultimatum.html", "ultimatum"), 
+    SJPN("prosecution", "sjpn.html", "sjpn");
+
+    private String assetId;
+    private String template;
+    private String resource;
+
+    ProsecutionType(String assetId, String template, String resource) {
+        this.assetId = assetId;
+        this.template = template;
+        this.resource = resource;
+    }
+
+    public String getAssetId() {
+        return assetId;
+    }
+
+    public void setAssetId(String assetId) {
+        this.assetId = assetId;
```

Comment:
These setters are not required and have been identified in Sonar as vulnerabilities. Please remove them.

Reply:
Done